crates/aptos/src/genesis/keys.rs

```diff
@@ -271,3 +273,45 @@ impl CliCommand<()> for GenerateLayoutTemplate {
         )
     }
 }
+
+/// Generate a WriteSet genesis compiled from a script file.
+///
+/// This will compile a piece of Move script and generate a writeset from that script.
+#[derive(Parser)]
+pub struct GenerateAdminWriteSet {
+    /// Path of the output genesis file
+    #[clap(long, parse(from_os_str))]
+    pub(crate) output_file: PathBuf,
+
+    #[clap(flatten)]
+    pub(crate) compile_proposal_args: CompileScriptFunction,
+
+    #[clap(long)]
+    pub(crate) execute_as: AccountAddressWrapper,
```

Comment:
Nit, put all the flattens at the bottom.  All other arguments need to have doc comments, they show up in the CLI help